tests/test_core.py

```diff
@@ -938,6 +940,101 @@ def test_scaper_instantiate():
         _compare_scaper_jams(jam, regjam)
 
 
+def create_scaper_with_random_seed(seed):
```

Comment:
rename: `_create_scaper_with_random_seed` (as this function if for internal use only)